core/domain/platform_parameter_domain_test.py

```diff
@@ -0,0 +1,86 @@
+# coding: utf-8
+#
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for the config variable registry."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+from core.domain import platform_parameter_domain as parameter_domain
+from core.tests import test_utils
+import utils
+
+
+class PlatformParameterChangeTests(test_utils.GenericTestBase):
+    VALID_CMD_NAME = (
+        parameter_domain
+        .PlatformParameterChange
+        .CMD_REPLACE_PARAMETER_RULES)
+
+    def test_platform_parameter_change_object_with_missing_cmd(self):
```

Comment:
What's the expected outcome here? Test names should outline both the conditions being tested and the expected outcome. E.g.:

``test_platform_param_object_with_missing_cmd_raises_exception``

Ditto elsewhere.

Reply:
Done.

Reply:
Please also follow this format for all other new tests. In general, test names should include the action being performed, the conditions under which that action is being performed, and the expected outcome. This helps code reviewers & other code maintainers more easily verify the intent of a test & whether it's correct by comparing the implementation with the test name. It also helps with auditing the tests for behavioral coverage, and better understand what's broken when the test is reported as failing.

Reply:
Ok, will ensure the format is applied for new tests in the following PRs.
(For tests in this PR, I think they are okay.)